kit.yml

```diff
@@ -29,3 +29,7 @@ certificates:
 credentials:
   base:
     agent: ssh 2048 fixed
+
+  secure:
+    failsafe:
```

Comment:
Why 'failsafe'?

Reply:
Because that's the name of the property in the manifest.  I could just call it admin_password, don't really care...

Reply:
Ok we'll address inconsistent design items in a separate workstream.